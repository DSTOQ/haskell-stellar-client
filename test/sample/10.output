TransactionEnvelope
  { _transaction =
      Transaction
        { _sourceAccount =
            GCRSOUMYGNYJO6OMHG3EH542J2RTFKFYPXLBCEBVGGIJOJJWHKK62N6R
        , _fee = FeeStroops { _feeStroops = 500 }
        , _seqNum =
            SequenceNumber
              { _sequenceNumber = NonNegativeInt64 (Refined 46283168877117510) }
        , _timeBounds = Nothing
        , _memo = MemoNone
        , _operations =
            [ Operation
                { _sourceAccount = Nothing
                , _body =
                    ManageOffer
                      ManageOfferOp
                        { _selling =
                            AssetCreditAlphanum
                              CNY GANJGJBGLCWPQGHBDS5AQN2YJ5DVPERFIGTWO7UZFMQSC5VBT3AOGH6P
                        , _buying = AssetNative
                        , _amount = NonNegativeInt64 (Refined 90000000)
                        , _price = Price { _numerator = 921 , _denominator = 2000 }
                        , _offerId = OfferId { _offerId = 0 }
                        }
                }
            , Operation
                { _sourceAccount = Nothing
                , _body =
                    ManageOffer
                      ManageOfferOp
                        { _selling =
                            AssetCreditAlphanum
                              USD GANJGJBGLCWPQGHBDS5AQN2YJ5DVPERFIGTWO7UZFMQSC5VBT3AOGH6P
                        , _buying = AssetNative
                        , _amount = NonNegativeInt64 (Refined 90000000)
                        , _price = Price { _numerator = 1747 , _denominator = 10000 }
                        , _offerId = OfferId { _offerId = 0 }
                        }
                }
            , Operation
                { _sourceAccount = Nothing
                , _body =
                    ManageOffer
                      ManageOfferOp
                        { _selling = AssetNative
                        , _buying =
                            AssetCreditAlphanum
                              XRP GANJGJBGLCWPQGHBDS5AQN2YJ5DVPERFIGTWO7UZFMQSC5VBT3AOGH6P
                        , _amount = NonNegativeInt64 (Refined 70000000)
                        , _price = Price { _numerator = 4333 , _denominator = 5000 }
                        , _offerId = OfferId { _offerId = 0 }
                        }
                }
            , Operation
                { _sourceAccount = Nothing
                , _body =
                    ManageOffer
                      ManageOfferOp
                        { _selling = AssetNative
                        , _buying =
                            AssetCreditAlphanum
                              XYG GANJGJBGLCWPQGHBDS5AQN2YJ5DVPERFIGTWO7UZFMQSC5VBT3AOGH6P
                        , _amount = NonNegativeInt64 (Refined 50000000)
                        , _price = Price { _numerator = 6917 , _denominator = 10000 }
                        , _offerId = OfferId { _offerId = 0 }
                        }
                }
            , Operation
                { _sourceAccount = Nothing
                , _body =
                    ManageOffer
                      ManageOfferOp
                        { _selling = AssetNative
                        , _buying =
                            AssetCreditAlphanum
                              XLM GANJGJBGLCWPQGHBDS5AQN2YJ5DVPERFIGTWO7UZFMQSC5VBT3AOGH6P
                        , _amount = NonNegativeInt64 (Refined 10000000)
                        , _price = Price { _numerator = 5897 , _denominator = 10000 }
                        , _offerId = OfferId { _offerId = 0 }
                        }
                }
            ]
        }
  , _signatures =
      [ DecoratedSignature
          { _hint = SignatureHint { _signatureHint = "363a95ed" }
          , _signature =
              Signature
                { _signature =
                    "871122916a649b171f093b883c4ed4ead5011294cab1a7fb46d56f07832323dd930e391629b48084106da89d6f3c30baff675cc0f06c3c8fcffb86422de1e10b"
                }
          }
      ]
  }