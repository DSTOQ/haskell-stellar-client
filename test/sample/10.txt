AAAAAKMnUZgzcJd5zDm2Q/eaTqMyqLh91hEQNTGQlyU2OpXtAAAB9ACkbkwAAABGAAAAAAAAAAAAAAAFAAAAAAAAAAMAAAABQ05ZAAAAAAAakyQmWKz4GOEcugg3WE9HV5IlQadnfpkrISF2oZ7A4wAAAAAAAAAABV1KgAAAA5kAAAfQAAAAAAAAAAAAAAAAAAAAAwAAAAFVU0QAAAAAABqTJCZYrPgY4Ry6CDdYT0dXkiVBp2d+mSshIXahnsDjAAAAAAAAAAAFXUqAAAAG0wAAJxAAAAAAAAAAAAAAAAAAAAADAAAAAAAAAAFYUlAAAAAAABqTJCZYrPgY4Ry6CDdYT0dXkiVBp2d+mSshIXahnsDjAAAAAAQsHYAAABDtAAATiAAAAAAAAAAAAAAAAAAAAAMAAAAAAAAAAVhZRwAAAAAAGpMkJlis+BjhHLoIN1hPR1eSJUGnZ36ZKyEhdqGewOMAAAAAAvrwgAAAGwUAACcQAAAAAAAAAAAAAAAAAAAAAwAAAAAAAAABWExNAAAAAAAakyQmWKz4GOEcugg3WE9HV5IlQadnfpkrISF2oZ7A4wAAAAAAmJaAAAAXCQAAJxAAAAAAAAAAAAAAAAAAAAABNjqV7QAAAECHESKRamSbFx8JO4g8TtTq1QESlMqxp/tG1W8HgyMj3ZMOORYptICEEG2onW88MLr/Z1zA8Gw8j8/7hkIt4eEL
