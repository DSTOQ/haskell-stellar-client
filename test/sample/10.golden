TransactionEnvelope {transaction = Transaction {sourceAccount = PublicKeyEd25519 {unPublicKeyEd25519 = 73796464499382006151152614252603807196781827224650109053774855681839325681133}, fee = Fee {unFee = 500}, seqNum = SequenceNumber {unSequenceNumber = 46283168877117510}, timeBounds = Nothing, memo = MemoNone, operations = VarLen {unVarLen = [Operation {sourceAccount = Nothing, body = ManageOffer (ManageOfferOp {selling = AssetCreditAlphanum4 (AssetCode4 {unAssetCode4 = 1129208064}) (PublicKeyEd25519 {unPublicKeyEd25519 = 12020110078372443445778830331393468099140010802731689699023048901958341411043}), buying = AssetNative, amount = 90000000, price = Price {numerator = 921, denominator = 2000}, offerId = OfferId {unOfferId = 0}})},Operation {sourceAccount = Nothing, body = ManageOffer (ManageOfferOp {selling = AssetCreditAlphanum4 (AssetCode4 {unAssetCode4 = 1431520256}) (PublicKeyEd25519 {unPublicKeyEd25519 = 12020110078372443445778830331393468099140010802731689699023048901958341411043}), buying = AssetNative, amount = 90000000, price = Price {numerator = 1747, denominator = 10000}, offerId = OfferId {unOfferId = 0}})},Operation {sourceAccount = Nothing, body = ManageOffer (ManageOfferOp {selling = AssetNative, buying = AssetCreditAlphanum4 (AssetCode4 {unAssetCode4 = 1481789440}) (PublicKeyEd25519 {unPublicKeyEd25519 = 12020110078372443445778830331393468099140010802731689699023048901958341411043}), amount = 70000000, price = Price {numerator = 4333, denominator = 5000}, offerId = OfferId {unOfferId = 0}})},Operation {sourceAccount = Nothing, body = ManageOffer (ManageOfferOp {selling = AssetNative, buying = AssetCreditAlphanum4 (AssetCode4 {unAssetCode4 = 1482245888}) (PublicKeyEd25519 {unPublicKeyEd25519 = 12020110078372443445778830331393468099140010802731689699023048901958341411043}), amount = 50000000, price = Price {numerator = 6917, denominator = 10000}, offerId = OfferId {unOfferId = 0}})},Operation {sourceAccount = Nothing, body = ManageOffer (ManageOfferOp {selling = AssetNative, buying = AssetCreditAlphanum4 (AssetCode4 {unAssetCode4 = 1481395456}) (PublicKeyEd25519 {unPublicKeyEd25519 = 12020110078372443445778830331393468099140010802731689699023048901958341411043}), amount = 10000000, price = Price {numerator = 5897, denominator = 10000}, offerId = OfferId {unOfferId = 0}})}]}}, signatures = VarLen {unVarLen = [DecoratedSignature {hint = SignatureHint {unSignatureHint = 909809133}, signature = Signature {unSignature = VarLen {unVarLen = "\135\DC1\"\145jd\155\ETB\US\t;\136<N\212\234\213\SOH\DC2\148\202\177\167\251F\213o\a\131##\221\147\SO9\SYN)\180\128\132\DLEm\168\157o<0\186\255g\\\192\240l<\143\207\251\134B-\225\225\v"}}}]}}